Trumpf.Coparoo.Web/Controls/Input.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Trumpf.Coparoo.Web.Controls
+{
+    using OpenQA.Selenium;
+
+    /// <summary>
+    /// Text input control object.
+    /// Expects an input html element with attribute type="text".
+    /// </summary>
+    public class Input : ControlObject
```

Comment:
Please rename the control to `TextInput`, there are many other `Input` controls. For a `TextInput` it's also clear why there is a text property, which is not the case for all the other types.

Reply:
What do you think, should we add an abstract base class for all the `input`-control objects, like this one, the `Checkbox` etc. or, alternatively move all of them into a separate namespace like `Controls.Input`? If we do the latter, then we could rename the class to simply `Text`.